.gitignore

```diff
@@ -9,3 +9,5 @@ plikHM_TTTEEE_lowl_lowE_lensing.tar.gz
 plikHM_TTTEEE_lowl_lowE_lensing/
 plikHM_TTTEEE_lowl_lowE_lensing_NS.tar.gz
 plikHM_TTTEEE_lowl_lowE_lensing_NS/
+.idea/*
```

Comment:
What are ``.idea/`` files?

Reply:
Those come with the PyCharm IDE which I use for cleaning up a project (nicely flags pep8 stuff...). Can just keep it locally though, if you don't want that.

Reply:
There shouldn't be any problem with keeping that in the ``.gitignore``